lib/ansible/modules/windows/win_pagefile.py

```diff
@@ -0,0 +1,156 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# (c) 2017, Liran Nisanov <lirannis@gmail.com>
+#
+# This file is part of Ansible
+#
+# Ansible is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# Ansible is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with Ansible.  If not, see <http://www.gnu.org/licenses/>.
+
+# this is a windows documentation stub.  actual code lives in the .ps1
+# file of the same name
+
+ANSIBLE_METADATA = {'metadata_version': '1.0',
+                    'status': ['preview'],
+                    'supported_by': 'community'}
+
+
+DOCUMENTATION = r'''
+---
+module: win_pagefile
+version_added: "2.4"
+short_description: Query or change pagefile configuration
+description:
+    - Query current pagefile configuration
+    - Enable/Disable AutomaticManagedPagefile
+    - Create new or override pagefile configuration
+options:
+  drive:
+    description:
+      - The drive of the pagefile.
+    required: false
+  initial_size:
+    description:
+      - The initial size of the pagefile.
+  maximum_size:
+    description:
+      - The maximum size of the pagefile.
+  override:
+    description:
+      - Override the current pagefile on the drive.
+    choices:
+      - true
+      - false
+    default: true
```

Comment:
So booleans require the following statements instead if the above:
```yaml
  type: bool
  default: 'yes'
```
And this will do the required to ensure the documentation is consistent.
We prefer 'yes' and 'no' for booleans as it is the most human readable format.
(And we have to make it strings, otherwise YAML will make it True/False again)